pymc3/sampling.py

```diff
@@ -631,6 +632,51 @@ def sample(
         return trace
 
 
+def subsample(
+    draws=1,
+    step=None,
+    start=None,
+    trace=None,
+    tune=0,
+    model=None,
+    random_seed=None,
+    callback=None,
+    **kwargs
+):
+    """A stripped down version of sample(), which is called only
+    by the RecursiveDAProposal (which is the proposal used in the MLDA stepper).
+    It uses some all necessary arguments and checks for the context in which it
```

Comment:
Is it "some" or "all"?

Reply:
This should be "some", will correct.

Reply:
@brandonwillard  Fixed in [17a4f39](https://github.com/alan-turing-institute/pymc3/commit/06e43e9464c9541624c783cf046c09c2e42fbd5d)